avogadro/rendering/scene.h

```diff
@@ -131,10 +131,26 @@ class AVOGADRORENDERING_EXPORT Scene
   /** Clear the scene of all elements. */
   void clear();
 
+  /**
+   * <<API Extension for TDX>>
+   * Calculte and return bounding box of the scene objects.
+   * @param minX [out] minimum X coordinate of the box diagonal
+   * @param minY [out] minimum Y coordinate of the box diagonal
+   * @param minZ [out] minimum Z coordinate of the box diagonal
+   * @param maxX [out] maximum X coordinate of the box diagonal
+   * @param maxY [out] maximum Y coordinate of the box diagonal
+   * @param maxZ [out] maximum Z coordinate of the box diagonal
+   * @param flags [in] flags informing which atoms will be included
+   * in the bounding box
+   */
+  void getBoundingBox(double& minX, double& minY, double& minZ, double& maxX,
```

Comment:
Again, I think it would be easier to access the bounding box directly from the molecule (which could adjust as needed for isosurface meshes, etc.)